src/lib/client.js

```diff
@@ -111,8 +111,8 @@ export default class Client {
   }
 
   // charges is a charges object
-  get_pdp_display(charges) {
-    const url = pdpHost(marketingUI, this.platform) + '/pdp/' + this.key + '/' + [charges.data.total, charges.data.shipping, charges.data.tax, charges.data.discount_amount, charges.data.grand_total].join(',');
+  get_pdp_display(charges, desktop = "center", mobile = "center") {
+    const url = pdpHost(marketingUI, this.platform) + '/pdp/' + this.key  + '/' + desktop + '/' + mobile + '/' + [charges.data.total, charges.data.shipping, charges.data.tax, charges.data.discount_amount, charges.data.grand_total].join(',');
```

Comment:
just a thought, is there any particular reason why this kind of thing couldn't be a standard GET param? something like `/pdp/:key?desktop=center&mobile=center` just curious if that would make maintenance easier vs having to update the URL path for each thing

Reply:
@widox - Reviewed your suggestion with Eben and after some consideration, he confirmed that we want to be very explicit with these params.